### PR TITLE
Correcting typo for multiple IfcSurfaceFeasture in IfcRelAdheres

### PR DESCRIPTION
**Pull request by SergejMuhic**
Schema was invalid :worried:

### CODE REVIEW

**Review by AlexBrad1eyCT — APPROVED** [review]
Damn it thought I got all the multiplicity changes! Nice catch

**Comment by SergejMuhic** [comment]
Blame the steptools compiler. I am merely the messenger. 😄

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcStructuralElementsDomain/Entities/IfcSurfaceFeature/DocEntity.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		<DocAttribute Name="PredefinedType" UniqueId="324c2136-9ea8-40b8-b332-c28ff31d3c3a" DefinedType="IfcSurfaceFeatureTypeEnum" AttributeFlags="1">
 			<Documentation>Indicates the kind of surface feature.</Documentation>
 		</DocAttribute>
-		<DocAttribute Name="AdheresToElement" UniqueId="290dfdce-d513-4a4a-844d-792db7e07217" DefinedType="IfcRelAdheresToElement" Inverse="RelatedSurfaceFeature">
+		<DocAttribute Name="AdheresToElement" UniqueId="290dfdce-d513-4a4a-844d-792db7e07217" DefinedType="IfcRelAdheresToElement" Inverse="RelatedSurfaceFeatures">
```
Comment (jmirtsch) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcStructuralElementsDomain/Entities/IfcSurfaceFeature/DocEntity.xml:12]: I agree, shouldn't also an inverse attribute be a SET ?

Comment (SergejMuhic) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcStructuralElementsDomain/Entities/IfcSurfaceFeature/DocEntity.xml:12]: In the case of a 1:1 this is the rule for inverse:
Entity data type: The inverse attribute contains exactly that one instance of the referenc-
ing entity data type which uses the current instance in the specied role. The cardinality of
the inverse relationship is 1 : 1 in this case.

See also:
IfcFeatureElementSubtraction.VoidsElements

**Pull request merged**